src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/WorkType.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.entities;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.io.Serializable;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@EqualsAndHashCode
+@Getter
```

Comment:
Better to just have the getter methods than use this annotation

Reply:
Done